src/noise_module.py

```diff
@@ -1331,31 +1331,30 @@ def taper(data):
             data[ii] *= win
     return data
 
-
-@jit(nopython = True)
-
+#jit(nopython = True)
 def moving_ave(A, N): ## change the moving average calculation to take as input N the full window length to smooth
     '''
     Alternative function for moving average for an array.
     PARAMETERS:
     ---------------------
-    A: 1-D array of data to be smoothed
```

Comment:
@niyiyu , are these changes needed given that you added `moving_ave_2D` in [this commit](https://github.com/mdenolle/NoisePy/commit/854cefa37625eabef7d3cb12166332b7e85e1ff4)?

Reply:
Yeah since I've had the 2D moving average added (sorry directly in the main branch. my bad). And original `moving_ave` should only take care of 1D case. We may not change this function.